spec/web_display_fake.rb

```diff
@@ -0,0 +1,61 @@
+class WebDisplayFake
```

Comment:
seems like this fake is pretty much the web_display? Or there is at least some duplication between the two.